scipy/stats/_continuous_distns.py

```diff
@@ -970,6 +975,73 @@ def _entropy(self):
 expon = expon_gen(a=0.0, name='expon')
 
 
+## Exponentially Modified Gaussian (exponential distribution
+##  convolved with a Gaussian)
+class expongauss_gen(rv_continuous):
+    """An exponentially modified Gaussian continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `expongauss` is::
+
+        expongauss.pdf(x, K) = 1/(2*K) exp(1/(2 * K**2)) exp(-x / K) * erfc(- (x - 1/K) / sqrt(2))
+           
+    where the shape parameter ``K`` > 0.
+
+    It can be thought of as the sum of a normally distributed random
+    value with mean ``loc`` and sigma ``scale`` and an exponentially
+    distributed random number with a pdf proportional to exp(-lambda * x)
+    where lambda = (``K`` * ``scale``)**(-1).
+
+    .. versionadded:: 0.16.0
+
+    %(example)s
+
+    """
+    def _argcheck(self, K):
+        return (K > 0)
+    
+    def _rvs(self, K, loc=0, scale=1):
+        invlam = scale * K
+        expval = self._random_state.standard_exponential(self._size) * invlam
+        gval = loc + scale * self._random_state.standard_normal(self._size)
+        return expval + gval
+
+    def _pdf(self, x, K):
+        invK = 1.0 / K
+        exparg = 0.5 * invK**2 - invK * x
+        # Avoid overflows; setting exp(exparg) to the max float works
+        #  all right here
+        expval = _lazywhere_single(exparg < _maxexparg, (exparg),
+                                   exp, _maxfloat)
```

Comment:
_lazywhere should work with a single argument: just feed it a single-element tuple. See, for example, invgamma.stats method.